src/ipc/ipc4/logging.c

```diff
@@ -10,12 +10,16 @@
 #include <ipc4/base_fw.h>
 #include <ipc4/error_status.h>
 #include <ipc4/logging.h>
+#include <zephyr/logging/log_backend.h>
+#include <zephyr/logging/log.h>
```

Comment:
we group headers alphabetically and by directories, not by function. Please leave them together with other Zephyr headers

Reply:
I think is to avoid the CONFIG_LOG_BACKEND_ADSP_MTRACE ifdef.